index.js

```diff
@@ -14,6 +14,7 @@ const CONFIRMATIONS = parseInt(process.env.CONFIRMATIONS || "3")
 const NODE_URL = process.env.NODE_URL || 'http://litecoind.default.svc.cluster.local:9332'
 const RPC_USERNAME = process.env.RPC_USERNAME || 'rpcuser'
 const RPC_PASSWORD = process.env.RPC_PASSWORD || 'rpcpassword'
+const EXPORT_TIMEOUT_MLS = parseInt(process.env.EXPORT_TIMEOUT_MLS || 1000 * 60 * 5)     // 5 minutes
```

Comment:
I think we should set this default to something like 15min, as the BTC block time is 10 min and this is the blockchain with the largest block time. For the other blockchains we can override the default and lower it, if we want to.